plugins/inputs/riemann_listener/riemann_listener.go

```diff
@@ -17,13 +17,13 @@ import (
 
 	"github.com/influxdata/telegraf/metric"
 
-	"github.com/gogo/protobuf/proto"
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
 	tlsint "github.com/influxdata/telegraf/plugins/common/tls"
 	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/plugins/inputs/riemann_listener/riemangoProto"
 	riemanngo "github.com/riemann/riemann-go-client"
-	riemangoProto "github.com/riemann/riemann-go-client/proto"
```

Comment:
Can you please either fork that repo or submit a PR upstream to update the generated protobuf?

Reply:
I have done a `replace` to use a fork of the `riemann-client-go` repo, and am going to submit a PR to that repo with the changes. I will PR to Telegraf if that PR is merged